articles/introduction-to-lxd-container-manager/index.md

```diff
@@ -0,0 +1,38 @@
+### Introduction
+LXD is a highly effective container manager. It provides an individual experience same as that offered by virtual machines. LXD systems are unique because they operate with Linux containers. This technology was introduced by Canonical Ltd alongside other contributors. LXD is supported by numerous Linux distributions. One of the significant pillars of LXD is a powerful REST API. 
+
+### Goal
+This article discusses the key features of LXD, the benefits, as well as its releases. 
+
+### Understanding what LXD entails
+LXD is a lightweight container manager which operates in the same way as a virtual machine. It does not require a Linux kernel and CPU hardware virtualization. LXD containers are more advantageous as compared to virtual machines. This is because the server can support more processes than virtual containers. LXD offers a hypervisor that helps to manage the life cycle of containers effectively. Furthermore, LXD containers are easy to control since all operations are done via a REST API. This means that accessing a remote server or the local host is seamless.
+
+### Key features of LXD
+LXD Containers are image-based. This is effectively achieved by a variety of Linux distributions worldwide. Linux containers possess great storage management. This is ideal for its multiple storage, pools, and volumes. Another feature is the device passthrough. Some of these components include block devices, Unix character paths, discs, NICs, GPU, and USB. Linux containers perform well in network management. The key activities in network management include configuration, creation of a network bridge, and the use of cross-host tunnels.
```

Comment:
`Some of these components include block devices, Unix character paths, discs, NICs, GPU, and USB.`

Specify links for these.

Reply:
What is `image-based`? How is it different from Docker? Brief about it